composer.json

```diff
@@ -19,7 +19,7 @@
   "require": {
     "php": ">=7.3",
     "ext-dom": "*",
-    "elan-ev/opencast-api": "1.7"
+    "elan-ev/opencast-api": "1.9.0"
```

Comment:
Should we fix that to 1.9 instead of 1.9.0? So we would get bugfixing releases easier

Reply:
Since I'm documenting the library upgrades through issues to ensure nothing gets overlooked, it's safe to leave it as is.